xml/obs_best_practice_kiwi_editor.xml

```diff
@@ -9,45 +9,165 @@
+      </para>
+     </listitem>
+    </itemizedlist>
+  </para>
+  </sect2>
+  <sect2>
+  <title>Adding Repositories in the &kiwi; Editor</title>
+  <para> To add a new repository you only need click to <emphasis>Add repository</emphasis> link and fill the dialog.
+    There are two ways to create it:
+    <itemizedlist>
+      <listitem>
+       <para>
+          <emphasis>Basic Mode:</emphasis> Adding the name of a project will provide a list with the repositories from that project.
+       </para>
+      </listitem>
+    </itemizedlist>
+  </para>
   <figure>
-   <title>&kiwi; Editor. Edit screen</title>
+   <title>&kiwi; Adding a new repository - Basic Mode</title>
    <mediaobject>
     <imageobject>
-     <imagedata fileref="edit_kiwi_edit.png"/>
+     <imagedata fileref="kiwi_editor_repo_basic.png"/>
     </imageobject>
    </mediaobject>
   </figure>
-  <para> Modify the &kiwi; configuration as you want, and click the "Save" button
-   when you finish. </para>
- </sect2>
+  <itemizedlist>
+    <listitem>
+      <para>
+        <emphasis>Expert Mode:</emphasis> This mode provides you with a set of customizable parameters for creating a repository.
+      </para>
+      <itemizedlist>
+        <listitem>
+         <para>
+           <emphasis>Type:</emphasis> Valid options are <emphasis>rpm-md</emphasis> and <emphasis>apt-deb</emphasis>.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Priority:</emphasis> Repository priority for the given repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Alias:</emphasis> Alternative name for the configured repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Source Path:</emphasis> Define the repository path
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>User:</emphasis> Specifies a user name for the given repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Password:</emphasis> Specifies a password for the given repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Prefer License:</emphasis> The repository providing this attribute will be used primarily to install
+            the license tarball if found on that repository.
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Image Include:</emphasis> Specifies whether the given repository should be configured as a repository in the image
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+            <emphasis>Replaceable:</emphasis> This attribute should only be applied in the context of a boot image description.
+            The specified repository may be replaced by the repositories specified in the image description.
```

Comment:
It could be more clear that this field defines a repository. Maybe say:

**Defines a repository name which** may be replaced by the repositories specified in the image description. This attribute should only be applied in the context of a boot image description.